proxy/src/main/java/com/wavefront/agent/histogram/QueuingChannelHandler.java

```diff
@@ -48,8 +58,15 @@ private void ship() {
   }
 
   private void innerAdd(T t) {
-    synchronized (this) {
-      buffer.add(t);
+    if (histogramDisabled.get()) {
+      // if histogram feature is disabled on the server increment counter and log it ...
+      logger.info("Ingested point discarded because histogram feature is disabled on the server");
```

Comment:
This might be super chatty. is there a way we can only report it every once in a while?